src/sentry/runner/initializer.py

```diff
@@ -301,6 +299,13 @@ def initialize_app(config, skip_service_validation=False):
         version=settings.ASSET_VERSION,
     )
 
+    import django
+    if hasattr(django, 'setup'):
+        # support for Django 1.7+
+        django.setup()
+
+    bind_cache_to_option_store()
```

Comment:
id love to figure out how we stop this from erroring when the db hasn't yet been created

Reply:
This function itself errors? If so, it's some side effect from just an import. What's the stacktrace? And I can look.

Reply:
take a look at the output in my comments, its at the very top on a clean database

Reply:
> 01:41:17 [ERROR] sentry: option.failed-lookup (key=u'github.apps-install-url')

So this is just being accessed at a point in time when it shouldn't. I'll track this down. I think that's the only reason this would error otherwise.